script/Arcueid/Day 11/11_00_ARC11_1.txt

```diff
@@ -197,7 +198,8 @@ It's just like it always is.
 [sha:e84500d91ea39c10b3b09c0c125fdfb622e55162]{
 -- Page 13, Offset 29511.
 -- この時間を守るために、危険に身を晒してきた。
-I'd put my life on the line to protect these ordinary moments.
+//'I'd' more imples future rather than past, which is fine, if that is the goal 
+I put my life on the line to protect these ordinary moments.
```

Comment:
Nope, I'd internally written this as a contraction of "I had" not "I would". Past is correct.

Reply:
To be clear I think your change here was good (harder to misread as future)